apps/studio/src/components/CoreTabs.vue

```diff
@@ -607,6 +624,15 @@ export default Vue.extend({
     async click(tab) {
       await this.setActiveTab(tab)
 
+    },
+      handleAltNumberKeyPress(event) {
+      if (event.altKey) {
+        const pressedNumber = Number(event.key); // Convert keyCode to the corresponding number
+        console.log("Pressed key is ", pressedNumber)
```

Comment:
I know our code is littered with useless logs, but we've been trying to keep it under control recently. If you wouldn't mind removing this, that would be great.